.travis.yml

```diff
@@ -0,0 +1,151 @@
+
+    # ARM_MUSCA_B1
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (ARM_MUSCA_B1)"
+      env: TARGET_NAME=ARM_MUSCA_B1 PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (ARM_MUSCA_B1)"
+      env: TARGET_NAME=ARM_MUSCA_B1 PROFILE=debug
+
+    # K64F
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (K64F)"
+      env: TARGET_NAME=K64F PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (K64F)"
+      env: TARGET_NAME=K64F PROFILE=debug
+
+    # K66F
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (K66F)"
+      env: TARGET_NAME=K66F PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (K66F)"
+      env: TARGET_NAME=K66F PROFILE=debug
+
+    # NUCLEO_F429ZI
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (NUCLEO_F429ZI)"
+      env: TARGET_NAME=NUCLEO_F429ZI PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (NUCLEO_F429ZI)"
+      env: TARGET_NAME=NUCLEO_F429ZI PROFILE=debug
+
+    # FVP_MPS2_M0
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (FVP_MPS2_M0)"
+      env: TARGET_NAME=FVP_MPS2_M0 PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (FVP_MPS2_M0)"
+      env: TARGET_NAME=FVP_MPS2_M0 PROFILE=debug
+
+    # FVP_MPS2_M0P
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (FVP_MPS2_M0P)"
+      env: TARGET_NAME=FVP_MPS2_M0P PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (FVP_MPS2_M0P)"
+      env: TARGET_NAME=FVP_MPS2_M0P PROFILE=debug
+
+    # FVP_MPS2_M3
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (FVP_MPS2_M3)"
+      env: TARGET_NAME=FVP_MPS2_M3 PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (FVP_MPS2_M3)"
+      env: TARGET_NAME=FVP_MPS2_M3 PROFILE=debug
+
+    # FVP_MPS2_M4
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (FVP_MPS2_M4)"
+      env: TARGET_NAME=FVP_MPS2_M4 PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (FVP_MPS2_M4)"
+      env: TARGET_NAME=FVP_MPS2_M4 PROFILE=debug
+
+    # FVP_MPS2_M7
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (FVP_MPS2_M7)"
+      env: TARGET_NAME=FVP_MPS2_M7 PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (FVP_MPS2_M7)"
+      env: TARGET_NAME=FVP_MPS2_M7 PROFILE=debug
+
+    # GD32_F450ZI
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - release (GD32_F450ZI)"
+      env: TARGET_NAME=GD32_F450ZI PROFILE=release
+
+    - <<: *mbed-compile-test
+      name: "Compile PSA example - debug (GD32_F450ZI)"
+      env: TARGET_NAME=GD32_F450ZI PROFILE=debug
```

Comment:
It would be good to use ccache support here - Please refer - https://github.com/ARMmbed/mbed-tools/blob/master/.travis.yml

Reply:
Waiting for CI to rerun to see if ccache is working (the stats is printed at the end of each compilation)

Reply:
Triggering a rebuild to see if there's any cache hits.

Reply:
I'd suggest a specific cache name here

Reply:
Ah good point! I though CACHE_NAME was used by Mbed CLI v2 but it turns out to be a Travis feature. Will add it.

Reply:
Done